### PR TITLE
Log Level fixes

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: blog-service/2023-04-07-search.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 New - We're excited to announce automatic log level detection for Log Search queries, helping you to quickly identify anomalies without having to search through large volume of logs to find high severity issues.
 
-With this update, you can visualize and filter log-level distribution in the Histogram results and Messages table messages of specific log level(s) in the same view. [Learn more](/docs/beta/search-log-level).
+With this update, you can visualize and filter log-level distribution in the Histogram results and Messages table messages of specific log level(s) in the same view. [Learn more](/docs/search/get-started-with-search/search-page/log-level).
```
Comment (JainM6) [inline blog-service/2023-04-07-search.md:18]: I think breaking this sentence into two parts could make it easier to read and understand. For example, you could say: 
With this update, you can now visualize and filter log-level distribution in both the **Histogram** results and **Messages** table. This allows you to view messages of specific log level(s) in the same view.

File: docs/search/get-started-with-search/search-page/log-level.md [hunk 2]
```diff
@@ -17,13 +17,13 @@ This feature is in Beta. To participate, contact your Sumo Logic account executi
 When performing **Log Search** queries in the UI, you can visualize and filter log-level distribution in your Histogram results and Messages table view, helping you to:
```
Comment (JainM6) [inline docs/search/get-started-with-search/search-page/log-level.md:17]:
```suggestion
When performing **Log Search** queries in the UI, you can visualize and filter log-level distribution in your **Histogram** results and **Messages** table view, helping you to:
```

File: blog-service/2023-04-07-search.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
 
 NOTE: This feature is in Beta. To participate, contact your Sumo Logic account executive or our Support Team.
```
[the remark this answers is not in the document]
Comment (JainM6) [reply blog-service/2023-04-07-search.md:20]: Do we also write a note like this?

**Review by JainM6 — DISMISSED** [review]
Minor formatting fixes

File: blog-service/2023-04-07-search.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (kimsauce) [reply blog-service/2023-04-07-search.md:20]: Thanks for asking. We discovered while you were out that our standard admonition formatting (`:::note`...) doesn't render in the RSS feed (the whole purpose of release notes) and looks broken. So we've begun formatting it this way.

File: docs/search/get-started-with-search/search-page/log-level.md (hunk 2 above)
Comment (kimsauce) [reply docs/search/get-started-with-search/search-page/log-level.md:17]: thanks!

File: blog-service/2023-04-07-search.md (hunk 1 above)
Comment (kimsauce) [reply blog-service/2023-04-07-search.md:18]: will do

Comment (kimsauce) [inline blog-service/2023-04-07-search.md:18]:
```suggestion
With this update, you can now visualize and filter log-level distribution in both your **Histogram** results and **Messages** table. This allows you to view messages of specific log level(s) in the same view. [Learn more](/docs/search/get-started-with-search/search-page/log-level).
```

File: docs/search/get-started-with-search/search-page/log-level.md [hunk 4]
```diff
@@ -14,16 +14,16 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
 * Quickly identify anomalies
-* Drill down in the high severity logs quickly
+* Drill down quickly into high severity logs
 * Navigate through a large volume of logs
 * Filter the relevant logs in their troubleshooting workflows
 
 <details><summary>What are log levels?</summary>
 
-Sumo Logic detects six log levels out of the box: FATAL, ERROR, WARN, INFO, DEBUG, and TRACE. If we're unable to find one of these log levels in a message, it is categorized under the OTHERS bucket.
+Sumo Logic detects six log levels out of the box: CRITICAL, WARN, ERROR, INFO, DEBUG, and TRACE. If we're unable to find one of these log levels in a message, it is categorized under the OTHERS bucket.
```
Comment (kimsauce) [inline docs/search/get-started-with-search/search-page/log-level.md:26]:
```suggestion
Sumo Logic detects six log levels out of the box: FATAL, ERROR, WARN, INFO, DEBUG, and TRACE. If we're unable to find one of these log levels in a message, it is categorized under the OTHERS bucket.
```

**Pull request merged**